obspy/taup/tau.py

```diff
@@ -663,3 +728,174 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(source_depth, min_degree=10, max_degree=90, nrecs=1000,
+                    phases=['P', 'S'], model='iasp91', fig=None, ax=None):
+        """
+        Returns a travel time plot and any created axis instance of this
+        plot.
+
+        :param source_depth: Source depth in kilometers.
+        :type source_depth: float
+        :param min_degree: minimum distance from the source (in degrees) to a
+            plot travel times Defaults to ``10``.
+        :type min_degree: float
+        :param max_degree: maximum distance from the source (in degrees) to
+            plot travel times. Defaults to ``90``.
+        :type max_degree: float
+        :param nrecs: Number of points to plot. Defaults to ``1000``.
+        :type nrecs: int
+        :param phases: List of phase names which should be used within the
+            plot. Defaults to ``['P', 'PKP']``.
+        :type phases: list of str, optional
+        :param model: string containing the model to use. Defaults to 'iasp91'.
```

Comment:
\``"iasp"\`` to get it to render correctly. Also maybe just document it as "Model name."